src/main/resources/data/testdata.sql

```diff
@@ -18,4 +18,11 @@ VALUES
 INSERT INTO
 `user_permission` (`user_id`, `permission_id`)
 VALUES
-(1,1),(1,2),(2,1),(2,2)
\ No newline at end of file
+(1,1),(1,2),(2,1),(2,2);
+
+INSERT INTO
+`t_source` (`name`, `type`, `config`)
+VALUES
+('myS3_1', 'AWS_S3', 's3_token1'),
```

Comment:
will we support multiple configs for the same type of storage?

Reply:
To keep simple, probably it is not needed.